enterprise/server/backends/userdb/userdb.go

```diff
@@ -638,6 +793,13 @@ func (d *UserDB) UpdateGroupUsers(ctx context.Context, groupID string, updates [
 					groupID).Error; err != nil {
 					return err
 				}
+				if err := tx.Exec(`
+						DELETE FROM APIKeys
+						WHERE user_id = ? AND group_id = ?`,
+					update.GetUserId().GetId(),
```

Comment:
Do group API keys have a NULL or empty user_id value? A malicious user could pass an empty user ID here.

Reply:
Moved this check into https://github.com/buildbuddy-io/buildbuddy/pull/3380